Dapper.Tests.Performance/Benchmarks.cs

```diff
@@ -44,15 +43,20 @@ public Config()
         {
             Add(new MemoryDiagnoser());
             Add(new ORMColum());
+            Add(TargetMethodColumn.Method);
             Add(new ReturnColum());
-            Add(Job.Default
-                .WithUnrollFactor(BenchmarkBase.Iterations)
-                //.WithIterationTime(new TimeInterval(500, TimeUnit.Millisecond))
+            Add(StatisticColumn.Mean);
+            Add(StatisticColumn.StdDev);
+            Add(StatisticColumn.Error);
+            Add(BaselineScaledColumn.Scaled);
+            Add(Job.Dry
                 .WithLaunchCount(1)
                 .WithWarmupCount(0)
-                .WithTargetCount(5)
-                .WithRemoveOutliers(true)
+                .WithInvocationCount(50)
+                .WithIterationCount(BenchmarkBase.Iterations)
+                .WithMaxIterationCount(BenchmarkBase.Iterations)
+                .WithOutlierMode(OutlierMode.All)
             );
         }
     }
-}
```

Comment:
This should make it work:

```
Add(new DefaultOrderer(SummaryOrderPolicy.FastestToSlowest));
SummaryPerType = false;
```

Reply:
I don't see an `Add()` overload for `IOrderer`, am I missing a namespace somewhere? Had it working at the `.Run()` call site over in Program.cs, but happy to move here if it works better.

Reply:
@NickCraver Sorry, it should be `Set`